python/eos/plot/plotter.py

```diff
@@ -1298,7 +1310,31 @@ def __init__(self, plotter, item):
                     self.weights = np.exp(item['data']['log_weights'])
                 else:
                     self.weights = None
+            elif 'data-file' in item:
+                if 'variable' not in item:
+                    raise KeyError('no variable specificed')
 
+                dfname = item['data-file']
+                eos.info('   plotting KDE for "{}"'.format(dfname))
+                prefix = os.path.split(dfname)[-1]
```

Comment:
why -1 and not 1? Is that for safety somehow? Can there be more than 1 head as output of os.path.split?

Reply:
I had the wrong functionality of ``os.path.split`` in mind, i.e., splitting into a list of arbitrary length at every path separator. Luckily, it worked here. Taking the last element is anyway what I intended.